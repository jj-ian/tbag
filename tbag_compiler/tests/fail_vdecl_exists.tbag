/* Authors: Maria */
true{
	string a = "blah";
	string a = "huh";
}