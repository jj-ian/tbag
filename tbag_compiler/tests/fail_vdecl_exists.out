Fatal error: exception Failure("Variable with name a exists.")
